### PR TITLE
2.4.0.Final announcement

### PR DESCRIPTION
**Pull request by Ladicek**

### CODE REVIEW

File: src/posts/announce-2.4.0.Final.adoc [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+* [https://issues.jboss.org/browse/THORN-2256[THORN-2256]] REST Client 1.2 (Done)
+* [https://issues.jboss.org/browse/THORN-2259[THORN-2259]] OpenTracing 1.3 (Done)
+* [https://issues.jboss.org/browse/THORN-2260[THORN-2260]] OpenAPI 1.1 (Done)
+* [https://issues.jboss.org/browse/THORN-2340[THORN-2340]] update OpenShift REST Client Java to 7.0.0 (Done)
+
+=== Bug
+* [https://issues.jboss.org/browse/THORN-2287[THORN-2287]] Multi-module projects with files in the same directories, e.g. META-INF/beans.xml (Done)
+* [https://issues.jboss.org/browse/THORN-2307[THORN-2307]] build fails on JDK 11 (Done)
+* [https://issues.jboss.org/browse/THORN-2311[THORN-2311]] conflicting definitions of the org.eclipse.microprofile.restclient module (Done)
+* [https://issues.jboss.org/browse/THORN-2312[THORN-2312]] Runner fails with ZipException: duplicate entry: /WEB-INF/classes/META-INF/jandex.idx (Duplicate Issue)
```
Comment (michalszynkiewicz) [inline src/posts/announce-2.4.0.Final.adoc:165]: Maybe changing the message here to indicate that it fixes support for multi-module projects would be good?

Comment (Ladicek) [reply src/posts/announce-2.4.0.Final.adoc:165]: Autogenerated :-) But I know there's more fixes to Runner, do you want to write an extra section to the main announcement?

Comment (michalszynkiewicz) [reply src/posts/announce-2.4.0.Final.adoc:165]: I added a section about Runner fixes. I wanted to create a PR for it and instead I modified your branch, sorry

Comment (Ladicek) [reply src/posts/announce-2.4.0.Final.adoc:165]: That is absolutely OK and actually what I expected :-)

**Comment by sberyozkin** [comment]
Reads great

**Comment by Ladicek** [comment]
I take it @michalszynkiewicz and @sberyozkin approved :-)

**Pull request merged**